doc/tutorial.md

```diff
@@ -282,47 +287,93 @@ While defining a RNN architecture is done in an usual manner (similarly to CTC,
+        expansion-gamma: [Number of additional candidates in expanded hypotheses selection (int)]
+        expansion-beta: [Allowed logp difference for prune-by-value method (float, > 0)]
 
 Except for the default algorithm, performance and decoding time can be controlled through described parameters. A high value will increase performance but also decoding time while a low value will decrease decoding time but will negatively impact performance.
 
-IMPORTANT (temporary) note: ALSD, TSD and NSC have their execution time degraded because of the current batching implementation. We decided to keep it as if for internal discussions but it can be manually removed by the user to speed up inference. In a near future, the inference part for transducer will be replaced by our own torch lib.
+IMPORTANT (temporary) note: ALSD, TSD, NSC and mAES have their execution time degraded because of the current batching implementation. We decided to keep it as if for internal discussions but it can be manually removed by the user to speed up inference. In a near future, the inference will be handled at C++ level.
 
-The algorithm references can be found in [methods documentation](https://github.com/espnet/espnet/tree/master/espnet/nets/beam_search_transducer.py). For more information about decoding usage, refer to [vivos config examples](https://github.com/espnet/espnet/tree/master/egs/vivos/asr1/conf/tuning/transducer).
+### Additional notes
 
-Additional notes:
 - Similarly to CTC training mode, transducer does not output the validation accuracy. Thus, the optimum model is selected with its loss value (i.e., --recog_model model.loss.best).
-- There are several differences between MTL and transducer training/decoding options. The users should refer to `espnet/espnet/nets/pytorch_backend/e2e_asr_transducer.py` for an overview.
+- There are several differences between MTL and transducer training/decoding options. The users should refer to `espnet/espnet/nets/pytorch_backend/e2e_asr_transducer.py` for an overview and `espnet/espnet/nets/pytorch_backend/transducer/arguments` for all possible arguments.
+- Memory usage during training can be reduced with minor training time augmentation using efficient encoder and decoder output combination [Li et al. (2019)](https://arxiv.org/pdf/1909.12415.pdf). To enable the feature, `joint-memory-reduction` can be set to True in training config.
 - RNN-decoder pre-initialization using a LM is supported. The LM state dict keys (`predictor.*`) will be matched to AM state dict keys (`dec.*`).
 - Transformer-decoder pre-initialization using a transformer LM is not supported yet.
-- Transformer and conformer blocks within the same architecture part (i.e: encoder) is not supported yet.
-- Customizable architecture is a in-progress work and will be eventually extended to RNN. Please report any encountered error or usage issue.
+- Mixing transformer and conformer blocks within the same architecture part (i.e.: encoder) is not supported.
 
```

Comment:
The following descriptions are common about the transducer and CTC/attention modes, but it looks like it is under the transducer. Please adjust the level of the header appropriately